### ISSUE TITLE
Import Copied Commands script not working with native notifications

### ISSUE THREAD

**Issue by pa-0** [hluk/CopyQ#2894]
[On Windows] If native notifications are enabled, the command to import commands copied to clipboard does not appear to work correctly.  In short, the button a user must press to confirm the import only appears when CopyQ-implemented notifications are active.  

With native notifications active, the user is notified only that something was copied to the clipboard.

**Comment by hluk** [comment]
Yeah, this is unfortunate, but AFAIK, there is no way of checking if buttons are supported by native notifications.
